README.md

```diff
@@ -93,15 +93,29 @@ Add this to your `.vimrc` file:
 
 `autocmd FileType java set omnifunc=javacomplete#Complete`
 
+To enable import R.java for an android project
```

Comment:
You can enable importing R.java for an Android project by using the maven-android-plugin by adding the following to your vimrc: